### PR TITLE
fix1531 sync init value from charts for UI config usage

### PR DESCRIPTION
**Pull request by w13915984028**
Harvester will open setting of monitoring for user config from UI, it is necessary to have init values for all fields, avoid blank values in WebUI.

Those values are same with (helm) charts in Harvester now, different value will overwrite (helm) charts and finally be effective in the POD setting.

Harvester issue:
https://github.com/harvester/harvester/issues/1531

### CODE REVIEW

**Comment by w13915984028** [comment]
@DaiYuzeng @WuJun2016 
FYI, with this commit, the initial value of harvester-monitoring will be existing.

cc @guangbochen

**Comment by w13915984028** [comment]
vagrant failure:

"TASK [boot Harvester Node 0] ***************************************************",
        "changed: [localhost]",
        "",
        "TASK [wait for Harvester Node 192.168.0.30 to get ready] ***********************",
        "FAILED - RETRYING: [localhost]: wait for Harvester Node 192.168.0.30 to get ready (20 retries left).",
      ...
        "FAILED - RETRYING: [localhost]: wait for Harvester Node 192.168.0.30 to get ready (1 retries left).",
        "fatal: [localhost]: FAILED! => changed=false ",
        "  attempts: 20",
        "  elapsed: 3",
        "  msg: 'Status code was -1 and not [200]: Request failed: <urlopen error [Errno 113] No route to host>'",
        "  redirected: false",
        "  status: -1",
        "  url: https://192.168.0.30/",

TASK [Cleanup Harvester Vagrant VMs] *******************************************
changed: [localhost]

TASK [Check result] ************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "ERROR: [WARNING]: No inventory was parsed, only implicit localhost is available\n[WARNING]: provided hosts list is empty, only localhost is available. Note that\nthe implicit localhost does not match 'all'\n[DEPRECATION WARNING]: \"include\" is deprecated, use include_tasks/import_tasks \ninstead. This feature will be removed in version 2.16. Deprecation warnings can\n be disabled by setting deprecation_warnings=False in ansible.cfg.\n[DEPRECATION WARNING]: \"include\" is deprecated, use \ninclude_tasks/import_tasks/import_playbook instead. This feature will be \nremoved in version 2.16. Deprecation warnings can be disabled by setting \ndeprecation_warnings=False in ansible.cfg."}

PLAY RECAP *********************************************************************
localhost                  : ok=14   changed=10   unreachable=0    failed=1    skipped=3    rescued=0    ignored=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:262)
..

**Comment by w13915984028** [comment]
Manual install:
image

POD has expected resource setting
image

image

UI has none-empty values:
image

image

Set with new value:
image

image

**Comment by bk201** [comment]
@w13915984028 Vagrant test builds ISO in the worker, not fetching from the master. Could you amend commit and force push again? Thanks

**Comment by bk201** [comment]
@w13915984028 Vagrant test builds ISO in the worker, not fetching from the master. Could you amend commit and force push again? Thanks

File: pkg/config/templates/rancherd-13-monitoring.yaml [hunk 1]
```diff
@@ -31,8 +31,12 @@ bootstrapResources:
         prometheusSpec:
           evaluationInterval: 1m
           resources:
+            limits:
+              cpu: 1000m
+              memory: 3000Mi
             requests:
-              cpu: 500m
+              cpu: 750m
+              memory: 750Mi
```
Comment (guangbochen) [inline pkg/config/templates/rancherd-13-monitoring.yaml:39]: per [Rancher's monitoring doc](https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/configuration/helm-chart-options/#configuring-resource-limits-and-requests) I think we may need to update those values accordingly.
image

Comment (w13915984028) [reply pkg/config/templates/rancherd-13-monitoring.yaml]: updated in https://github.com/harvester/harvester-installer/commit/299c22aefde185ce98cade0d46499e86fe66a4b2

**Comment by w13915984028** [comment]
@bk201  it still failed in Vagrant installation testing, any idea? thanks.

cc @guangbochen 

I tried install a work node manually, it is successful.
image

**Comment by guangbochen** [comment]
Yep, this should be a CI-related issue and @bk201 is also checking the root cause, feel free to contribute a PR if u guys found the root causes, thanks.

**Review by guangbochen — APPROVED** [review]
LGTM

**Review by johnliu55tw — APPROVED** [review]
Verified UI shows those default values.

**Pull request merged**